code/game/objects/items/rogueitems/natural/wood.dm

```diff
@@ -34,7 +34,7 @@
 		lumber_amount = rand(minimum, max(round(skill_level), minimum))
 		var/essense_sound_played = FALSE //This is here so the sound wont play multiple times if the essense itself spawns multiple times
 		for(var/i = 0; i < lumber_amount; i++)
-			if(prob(skill_level+ user.goodluck(2)))
+			if(user.stat_roll(STATKEY_LCK,2,10))
```

Comment:
this is still just a straight nerf

Reply:
God dangit I changed it and then the revert screwed it